docs/tutorials/getting-started-flakes.md

```diff
@@ -12,6 +12,37 @@ project and its dependencies into Nix code.
 Assuming you have [Nix](https://nixos.org/download.html) installed, you can
 start setting up your project.
 
+## Using `flake init` and `nix`
+
+The `flake init` command create an example `hello` package from hackage
+containing an `flake.nix` and `nix/hix.nix` file. The project can be used with
+regular `nix` tools.
+
+```bash
+nix flake init --template haskell-nix#template --impure
```

Comment:
I think this should be `nix flake init --template templates#haskell.nix --impure`